LoopKitUI/Views/MultipleSelectionList.swift

```diff
@@ -25,8 +25,7 @@ public struct MultipleSelectionList<Item>: View where Item: Hashable {
                                        selectedItems: self.$selectedItems,
                                        itemToDisplayableString: self.itemToDisplayableString)
         }
-        .listStyle(GroupedListStyle())
-        .environment(\.horizontalSizeClass, .regular)
+        .insetGroupedListStyle()
```

Comment:
[nit] specific size class being used here. Is this accounted for?

Reply:
I'm guessing the reason this didn't use the `HorizontalClassOverride` is because it didn't exist yet.